### PR TITLE
Synthetix

### PR DESCRIPTION
**Pull request by zincoshine**
Added a new flag to the assets model and changed circleci pipeline to publish to npmjs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,9 +46,9 @@ jobs:
             npm version $(node -e "const currentVersion=require('./package.json').version; const firstTwoDots=currentVersion.substring(0, currentVersion.lastIndexOf('.')+1); console.log(firstTwoDots);")$CIRCLE_BUILD_NUM
       - run:
           name: Authenticate with registry
-          command: curl -u $ARTIFACTORY_PUBLISHING_USER:$ARTIFACTORY_PUBLISHING_PASSWORD https://pillarproject.jfrog.io/pillarproject/api/npm/auth >> ~/common-models/.npmrc
+          command: echo "//registry.npmjs.org/:_authToken=$PILLAR_NPM_TOKEN" > ~/common-models/.npmrc
```
Comment (antonioned) [inline .circleci/config.yml:49]: @vnovacev can we add the PILLAR_NPM_TOKEN to the project in circleci?

**Review by antonioned — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (vnovacev) [reply .circleci/config.yml:49]: sure, will do it right now

Comment (zincoshine) [reply .circleci/config.yml:49]: @antonioned  - Is that something @vnovacev will do on circleci or you want me to do?

Comment (vnovacev) [reply .circleci/config.yml:49]: `PILLAR_NPM_TOKEN` env var is added

**Review by vnovacev — APPROVED** [review]

**Review by antonioned — APPROVED** [review]

**Review by rafaelcidade — APPROVED** [review]

**Pull request merged**